US-CBP/cbp-theme
# Ds css refactor

#### What's this PR do?
- Added design tokens via style-dictionary in `ds-css` 
- Restructured files in `ds-css` for better legibility, removed redundant empty files
- Added separate file, `_vendor.scss`, to compartmentalize Bulma configurations
- Made `ds-css` scss files more modular & future-proof against eventual deprecation of @import by refactoring @import statements into @use/@forward
- Minor text corrections in iconography page

#### Where should the reviewer start?
- Files that need to be reviewed (please take some time to go over the changes if you can because it's a lot).

- `ds-css` --> `styles` - most changes are found here. 
- `ds-css` --> `config.json` - config for style-dictionary. 
- `ds-ux-guidelines` --> `ds-common-styles.scss` , `header.scss`, `navigation.scss` (is there a reason why only this file is using css variables?) - utilizes the @use rule to call variables and classes in `ds-css`.
- `ds-ux-guidelines` --> `layout-development.scss` - still has import statements for now, not sure if you can use @use for importing fonts. 

#### How should this be manually tested?
- install/build `ds-css` and `ds-ux-guidelines`, and make sure the style guide looks the same as the one in production. 

#### Any background context you want to provide?
-Please let me know if you have any questions, or if there are issues with compiling. Variable names/order are temporary and will be changed sometime in the future. 

#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
